conf.yaml

```diff
@@ -113,6 +113,9 @@ contents:
               -
                 repo:   stack-docs
                 path:   docs/en
+              -
+                repo:   elasticsearch
+                path:   docs
```

Comment:
Do you think you'd be ok with `docs/reference` or something like that? Making these as precise as possible will cut down on noop builds.

Reply:
Makes sense. Done with a04d63a.